DigitalLearningSolutions.Data/DataServices/SupervisorDelegateDataService.cs

```diff
@@ -69,6 +69,7 @@ IEnumerable<string> emails
                         sd.ID,
                         sd.SupervisorAdminID,
                         sd.SupervisorEmail,
+                        sd.CandidateID_deprecated,

```

Comment:
This doesn't appear to relate to this fix and is reinstating a deprecated field, can we revert this change?

Reply:
change reverted